install.sh

```diff
@@ -2,12 +2,34 @@
 set -euo pipefail
 IFS=$'\n\t'
 
+function welcome() {
```

Comment:
The colors for this could definitely be improved-- as it stands, they somewhat clash with eachother, and it might be better to try to replicate the normal cpt logo.
![image](https://user-images.githubusercontent.com/77183348/229523002-e4e7cd8e-24af-4f26-8460-c418c7dd6d55.png)

Reply:
but wouldn't that not work with terminals other than Kitty?

Reply:
oh lol, I'm stupid, lemme do that

Reply:
> No, I just mean the colors

ok, so I'm not gonna be able to do that, since ANSI limits me to like 6 colors (2 of which are not in the logo). what i can do is remove the colors from the logo and keep it plain

Reply:
Technically you can actually do rgb colors with escape[32;2;r;g;bm, https://gist.github.com/fnky/458719343aabd01cfb17a3a4f7296797 has a great explanation of everything, but you can probably use the default colors for the rainbow (and chances are if they are a cpt user their colorscheme will already be correct)

Reply:
(Escape being whatever escape sequence you prefer, I usually use \x1b)

Reply:
Oki!